Исправил в соответствии правилам git

adobryakovreon opened the pull request.

adobryakovreon committed:
Исправил в соответствии правилам git

adobryakovreon committed:
constant naming fix

KolesnichenkoAleksey commented on index.js:
```diff
@@ -15,20 +15,20 @@ app.use(express.urlencoded({ extended: true }));
 const BIRTHDAY_FIELD_ID = 376795;
+
 app.post("/create", async (req, res) => {
 		const {id} = req.body.contacts.add[0].id;
```
Можно деструктуризировать и достать id таким образом: [{id}]

KolesnichenkoAleksey commented on index.js:
```diff
@@ -15,20 +15,20 @@ app.use(express.urlencoded({ extended: true }));
-		const BIRTHDAY = utils.getFieldValue(CUSTOM_FIELDS, BIRTHDAY_FIELD_ID);
-		const AGE_VALUE = utils.getAge(BIRTHDAY);
-		const AGE_FIELD = utils.makeField(AGE_FIELD_ID, AGE_VALUE);
+		const custom_fields = req.body.contacts.add[0].custom_fields;
```
Точно так же, можно деструктуризировать

KolesnichenkoAleksey commented on index.js:
```diff
@@ -15,20 +15,20 @@ app.use(express.urlencoded({ extended: true }));
+		const birthday = utils.getFieldValue(custom_fields, BIRTHDAY_FIELD_ID);
+		const age_value = utils.getAge(birthday);
+		const age_field = utils.makeField(AGE_FIELD_ID, age_value);
 		req.body = {
```
Не очень хорошо закидывать в тело запроса данные, это данные, которые не должны меняться, лучше выделить отдельную переменную, например updatedContact

KolesnichenkoAleksey commented on utils.js:
```diff
@@ -0,0 +1,55 @@
+	};
+};
+
+const getAge = (birthday) => {
+	return birthday ? new Date(Date.now()).getFullYear() - Number(birthday.split(".")[2]): undefined;
+};
```
Функция работает некорректно, если вызвать функцию между текущей датой и датой следующего дня рождения, функция вернёт на один год меньше, так же, функция очень чувствительна к формату ввода даты(дд.мм.гггг), что очень не хорошо

adobryakovreon committed:
Исправил деструктуризацию, изменил логику getAge() - теперь дата приходит
 в Unix-формате, и проходит проверки на день и месяц, исходя из которых
 вычисляет полные года контакта. Обновленный контакт занес в отдельную переменную,
 тело запроса не трогаю.

adobryakovreon committed:
Почистил логи

KushchevR commented on index.js:
```diff
@@ -14,21 +14,23 @@ app.use(express.json());
-				age_field,
+				ageField,
 			]
 		};
```
Можно еще вынести в отдельный модуль и разнести логику

KolesnichenkoAleksey replied:
... а главное зачем
